### ISSUE TITLE
Print Quest Completion Dialogue to Chat Window

### ISSUE THREAD

**Issue by Nevicar** [Questie/Questie#3542]
## Description

If possible, I think that adding a check box under or around "Auto Complete" that offers "Print Completion Text" would be wonderful. I use auto complete just for ease and would appreciate having the quest completion dialogue printed out so that I can read it as I leave. AutoTurnIn for retail does this by default and it's very useful. 

Thanks for readin!

**Comment by BreakBB** [comment]
Hey @Nevicar to make sure we understand your request correct. You want to get the text that is shown in the NPC dialog before clicking "Complete" to turn in the quest, right?

**Comment by anon1231823** [comment]
We don't store this information in our database. So this just is never realistically going to be done.

**Comment by BreakBB** [comment]
I don't think we need this information in our database for the feature to work.
